compose/compose-file/compose-versioning.md

```diff
@@ -4,14 +4,24 @@ keywords: fig, composition, compose, versions, upgrading, docker
 title: Compose file versions and upgrading
 ---
 
+>**Warning**
+>
+>This page contains information on the legacy versions of Compose, also collectively referred to as Compose V1.
+>From the end of June 2023 Compose V1 won’t be supported anymore.
```

Comment:
@aevesdocker Is there a better way to say this? It feels a bit passive… But I am not sure of the internal processes behind this…

Reply:
This matches what we have elsewhere in the docs since Jan. But I hear your point. I'll maybe address this in a follow-up PR